src/main/interface/msp.c

```diff
@@ -924,7 +924,11 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
 #else
         sbufWriteU32(dst, 0);
 #endif
-        sbufWriteU16(dst, getEstimatedVario());
+#ifdef USE_VARIO
+        sbufWriteU16(dst, (int16_t)getEstimatedVario());
```

Comment:
What do you do in case of an int16 overflow?

Reply:
Constrained the result to fit within limit of 16 bit signed int.